ruby-gem/lib/calabash-android/operations.rb

```diff
@@ -92,6 +94,10 @@ def deprecated_actions
       ]
     end
 
+    def adb_command
```

Comment:
I exposed this so that we could use it from `drag_helpers`, but I don't think we want users to see this. How to encapsulate?